### PR TITLE
Installation and Usage

### PR DESCRIPTION
**Pull request by AlMaVizca**
Hi!

I'm happy for find this project, becouse I'm learning flask and vuejs too.

I did this changes to get the application running.
Dockerfile creates the working enviroment and also the database.

Please let me know if there is something that could be done in a better way.

### CODE REVIEW

File: flask-vue/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM python:latest
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+RUN apt-get update && \
+    apt-get install -y nodejs npm &&\
+    rm -rf /var/lib/apt/lists/*
```
Comment (italomaia) [inline flask-vue/Dockerfile:7]: What's the purpose of this line? "rm -rf /var/lib/apt/lists/*"

**Comment by italomaia** [comment]
Besides the comment above, this PR seems alright. Good to know it helps you. It still lacks a few improvements, like integration with gulp and vueify.

File: flask-vue/Dockerfile (hunk 1 above)
Comment (AlMaVizca) [inline flask-vue/Dockerfile:7]: The propose is reduce the size of a docker image.
That command deletes the lists of packages downloaded by apt-get update

**Comment by italomaia** [comment]
Nice! Thanks for the clarification.

**Pull request merged**